src/components/AvatarMenu/AvatarMenu.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import Avatar from "@mui/material/Avatar";
+import List from "@mui/material/List";
+import MenuItem from "@mui/material/MenuItem";
+import Menu from "@mui/material/Menu";
+import Box from "@mui/material/Box";
+import { useAuthContext } from "../../contexts/AuthContext";
+
+const options = ["Sign Out"];
+
+const AvatarMenu = () => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [selectedIndex, setSelectedIndex] = useState(1);
+
+  const history = useHistory();
+  const { signout } = useAuthContext();
+
+  const handleClickListItem = (event) => {
+    setAnchorEl(event.target);
+  };
+
+  const handleMenuItemClick = (index) => {
+    setSelectedIndex(index);
+    setAnchorEl(null);
+    signout(() => {
+      history.push("/");
+    });
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
```

Comment:
⛏️ It looks like there's an extra set of `div` tags here.

Reply:
Addressed: commit 1c3755dbcd379ff9772a77a70f14a4cba198d8d1